src/mentionPlugin/MentionSearch/index.js

```diff
@@ -48,6 +54,7 @@ export default (mentions) => {
               ))
             }
           </Dropdown>
+          : void 0}
```

Comment:
Is there are reason to use `void 0` instead of `undefined`? I think UglifyJs does the transform anyway and `void 0` is a bit confusing.